pillar.example

```diff
@@ -84,6 +84,12 @@ postgresql:
       database: production
       address: 10.1.2.0/24
       method: trust
+      #NOTE: The following values represent the defaults and are for documentation purposes only.
```

Comment:
Please add a space after the `#`.
I think the "documentation only" section can be removed, as the values are completely valid. 

```
# The following values represent the defaults
``

Would be sufficient.